session/src/main/resources/reference.conf

```diff
@@ -0,0 +1,12 @@
+alpakka.cassandra {
+  # The implementation of akka.cassandra.session.SessionProvider
+  # is used for creating the CqlSession. By default it loads all the driver's default
+  # settings. The driver allows to override the way configuration is loaded
+  # see: https://docs.datastax.com/en/developer/java-driver/4.3/manual/core/configuration/#quick-overview
+  # It may optionally have a constructor with an ActorSystem and Config parameter.
+  # The config parameter is this config section of the plugin.
+  session-provider = "akka.stream.alpakka.cassandra.DefaultSessionProvider"
+
+  # A name that uniquely identifies the Cassandra driver instance.
+  session-name = "alpakka"
```

Comment:
I suggest that we remove `session-name` and read that from the "standard" place `datastax-java-driver.basic.session-name`, to avoid confusion of having it in more places.
I did that in https://github.com/akka/akka-persistence-cassandra/pull/683